packages/fuels-accounts/src/provider.rs

```diff
@@ -366,15 +366,17 @@ impl Provider {
     pub async fn dry_run_opt(
         &self,
         tx: impl Transaction,
-        utxo_validation: bool,
+        utxo_validation: Option<bool>,
```

Comment:
Do we know the difference between `None` and `Some(false)`?

Reply:
I could not find one so i reverted to the old code.